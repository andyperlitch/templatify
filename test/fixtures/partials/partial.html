{{!
{
  "name" : "partial",
  "decription" : "A partial to test things out"
}
}}
<div class="partial">
  This is a partial include.

  {{> test_fixtures_partials_bis }}

</div>
