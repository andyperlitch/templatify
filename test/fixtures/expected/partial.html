<div class="best class ever">
  This plugin is my templatify plugin ever.

  
<div class="partial">
  This is a partial include.

  
<div class="partial">
  And another partial, yay. I should be nested and be happy.
</div>


</div>

</div>
